.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- Thank you for submitting this issue!
+-->
+
+**I'm submitting a ...**  (check one with "x")
+- [ ] bug report
```

Comment:
Please add a suboption here for "Error pages" (400,403,404,500)

Reply:
@Pl4gue What you mean ?

Reply:
Because I have a "report issue" link on the error pages :) (so that you can directly report the error if you e.g. get HTTP500)

Reply:
Gotcha! will update